### PR TITLE
MySQL and database

### PR DESCRIPTION
**Pull request by ValdasPiekus**
Added MySQL support to back-end, created a flyway baseline file which includes queries for the database

### CODE REVIEW

File: src/main/resources/db/migration/V1__Baseline.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
+CREATE TABLE tbl_Users_Credentials (
+login VARCHAR(30) NOT NULL PRIMARY KEY,
+password VARCHAR(254) NOT NULL,
+created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+created_by VARCHAR(30),
```
Comment (liuxess) [inline src/main/resources/db/migration/V1__Baseline.sql]: Default value for this and updated_by should be inserter's username

File: src/main/resources/db/migration/V1__Baseline.sql [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+updated TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+updated_by VARCHAR(30),
+PRIMARY KEY (login),
+FOREIGN KEY (login) REFERENCES tbl_Users_Credentials(login)
+);
+
+ALTER TABLE tbl_Users
+ADD COLUMN full_name VARCHAR(70)
```
Comment (liuxess) [inline src/main/resources/db/migration/V1__Baseline.sql]: first name 30 symbols... last name 30 symbols....full name - 70 symbols.... HMMMMMMM

File: src/main/resources/db/migration/V1__Baseline.sql [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+created_by VARCHAR(30),
+updated TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+updated_by VARCHAR(30)
+);
+
+CREATE TABLE tbl_Issues_Steps (
+issue_id INT(6) UNSIGNED NOT NULL,
+responsible VARCHAR(30) NOT NULL,
+step_id INT(6) UNSIGNED NOT NULL,
+comment VARCHAR(254) NOT NULL,
```
Comment (liuxess) [inline src/main/resources/db/migration/V1__Baseline.sql]: So when we proceed to the next step (thus creating it), what will the comment be?

**Review by liuxess — APPROVED** [review]

File: src/main/resources/db/migration/V1__Baseline.sql (hunk 3 above)
Comment (ValdasPiekus) [reply src/main/resources/db/migration/V1__Baseline.sql]: Not null was removed

**Pull request merged**